### PR TITLE
Added Signup and login page

### PR DESCRIPTION
**Pull request by Isha2806**
## Added Login page and sign up page
Screenshot (112)

## Features to avoid 
UI and responsiveness of the app. Will work on optimization once done with the functionalities.

### CODE REVIEW

File: src/pages/Login/login.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
+
+.signup-page{
+    background-color:#E6D5B8;
```
Comment (Asuraking1n) [inline src/pages/Login/login.css:3]: You should use color variables

File: src/pages/Login/login.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+.signup-page-content{
+position: absolute;
+  top: 14%;
+  left: 50%;
+  transform: translateX(-50%);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 0.1rem solid #cfcfcf;
+  padding: 3rem;
```
Comment (Asuraking1n) [inline src/pages/Login/login.css:18]: Why giving border in rem ? In one onf his sessions Vikram said that we should use px for properties like border, box shadow, border radius bcz we don't want to change these on screen size change

**Review by ArjunGTX — COMMENTED** [review]
Have suggested some changes. Please go through them

File: src/pages/Products/filter.jsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+                       <p>100000</p>
+                   </div>
+                   <input type="range"min="100" max="100000"/>
+               </div>
+           
+           <h1 className="filter-heading">Category</h1>
+           <ul className="side-list">
+               <li>
+                   <input type="checkbox" />
+                   <label for = "newCome">New Releases</label>
```
[the remark this answers is not in the document]
Comment (Isha2806) [reply src/pages/Products/filter.jsx:25]: Thankyou..did not knew about this. i have fixed it.

File: src/pages/Login/login.css (hunk 1 above)
Comment (Isha2806) [reply src/pages/Login/login.css:3]: fixed the color variables

File: src/pages/Login/login.css (hunk 2 above)
Comment (Isha2806) [reply src/pages/Login/login.css:18]: Thankyou! Fixed it

File: src/App.css [hunk 4]
```diff
@@ -1,4 +1,63 @@
...
+  position: absolute;
+  top: 0.4rem;
+  left: 0.5rem;
+}
+
+.blue-btn {
+  width: 35rem;
+  height: 5rem;
+  background-color: #036bb1;
+  color: #fff;
```
[the remark this answers is not in the document]
Comment (Isha2806) [reply src/App.css:46]: Fixed the color variables

File: src/components/Category/category.jsx [hunk 5]
```diff
@@ -1,14 +1,39 @@
...
     return(
         <div class = "product-category">
         <img class = "avatar bahut-hi-badka"
-        src = "https://unsplash.com/photos/6SHd7Q-l1UQ"
+        src = {footwear}
         alt = "Categories of gifts"/>
         <p>Birthday</p>
-
-     </div>
+        <img class = "avatar bahut-hi-badka"
```
[the remark this answers is not in the document]
Comment (Isha2806) [reply src/components/Category/category.jsx:12]: I will map it to products from backend. I have hardcoded it just for now. will fix this in next Prs.

**Pull request merged**